libs/features/unused-guid-search/src/unused-guid-search.service.ts

```diff
@@ -0,0 +1,89 @@
+import { Injectable, inject } from '@angular/core';
+import { MysqlQueryService } from '@keira/shared/db-layer';
+import { Observable, catchError, map, throwError } from 'rxjs';
+
+export const MAX_INT_UNSIGNED_VALUE = 4294967295;
+
+export type DbOptions = {
+  table: string;
+  key: string;
+  label: string;
+};
+
+@Injectable({
+  providedIn: 'root',
+})
+export class UnusedGuidService {
```

Comment:
Missing unit tests for this service

Reply:
@sogladev ok, however it would still be good to have a placeholder test even for just creating the `service` and asserting that it is defined. I know it might sound useless but sometimes this kind of tests can find issues, for example when one does big refactoring or other major technical changes.